Products/ZenUI3/browser/resources/js/zenoss/MIBs.js

```diff
@@ -234,6 +234,118 @@ node_details = new Ext.form.FormPanel({
+                renderTo: Ext.getBody()
+            },{
+                xtype: 'panel',
+                title: 'OID to add:',
+                html: '<p>Name: ' + addParams['id'] +'<br>'
+                        + 'OID: ' + addParams['oid'] +'<br>'
+                        + 'Organizer: ' + oidData['new_moduleName'] + '</p><br>',
+                renderTo: Ext.getBody()
+            },{
+                xtype: 'panel',
+                html: '<p><br>Do you want to continue this operation?</p><br>',
+                renderTo: Ext.getBody()
+            }],
+            buttons: [{
+                xtype: 'DialogButton',
+                text: _t('Yes'),
+                handler: function() {
+                    addParams['confirm_change'] = true;
+                    currentAddFn(addParams, function(response) {
+                        if (response.success) {
+                            if (response.data['different_container'] === false) {
+                                Zenoss.message.info(_t('OID was successfully added.'));
+                                Ext.getCmp('gridCardPanel').layout.activeItem.refresh();
+                            }
+                            else {
+                                Zenoss.message.info(_t('Problem occurred. Try again.'));
+                            }
+                        }
+                    });
+                }
+            }, Zenoss.dialog.CANCEL]
+        }]
+    });
+    dialog.show();
+}
+
+function createAddNodeDialog() {
+    var dialog = new Zenoss.dialog.CloseDialog({
+        id: 'addNodeDialog',
+        width: 300,
+        title: currentAddNodeTitle,
+        items: [{
+            xtype: 'form',
+            buttonAlign: 'left',
+            fieldDefaults: {
+                labelAlign: 'top'
+            },
+            footerStyle: 'padding-left: 0',
+            id: 'addNodeForm',
+            items: [{
+                fieldLabel: _t('ID'),
+                xtype: 'textfield',
+                allowBlank: false,
+                name: 'id'
+            },{
+                fieldLabel: _t('OID'),
+                xtype: 'textfield',
+                allowBlank: false,
+                name: 'oid'
+            }],
+            buttons: [{
+                xtype: 'DialogButton',
+                text: _t('Submit'),
+                handler: function() {
+                    var form = Ext.getCmp('addNodeForm').getForm();
+                    var addParams = {
+                            uid: Zenoss.env.currentUid,
+                            id: Ext.htmlEncode(form.findField('id').getValue()),
+                            oid: Ext.htmlEncode(form.findField('oid').getValue())
+                        };
+                    currentAddFn(addParams, function(response) {
+                        if (response.success) {
+                            if (response.data['different_container'] === true) {
```

Comment:
To match the meaning of `confirm_change`, this parameter should be named `confirmation_required`.